Sources/SmileID/Classes/SmileID.swift

```diff
@@ -265,6 +280,19 @@ public class SmileID {
     public class func apply(_ localizableStrings: SmileIDLocalizableStrings) {
         self.localizableStrings = localizableStrings
     }
+    
+    /// Set wrapper SDK information
+    /// This method should be called by wrapper SDKs to identify themselves
+    /// in the metadata collection.
+    /// - Parameters:
+    ///   - name: The name of the wrapper SDK (e.g., "Flutter" or "React Native")
+    ///   - version: The version of the wrapper SDK
+    class func setWrapperInfo(name: String, version: String) {
```

Comment:
@tobitech let's expose this function publicly and add some documentation that a partner shouldn't use it. You can follow https://github.com/smileidentity/android/blob/284b5d9724a6ab901193bf1581c49740ed1e911d/lib/src/main/java/com/smileidentity/SmileID.kt#L592